packages/esm-patient-notes-app/src/notes/visit-notes.resource.ts

```diff
@@ -106,7 +89,12 @@ export function saveVisitNote(abortController: AbortController, payload: VisitNo
     signal: abortController.signal,
   });
 }
-
-function getConceptReferenceTermCode(conceptMapping: Array<ConceptMapping>): ConceptMapping {
-  return conceptMapping.find((concept) => concept.conceptReferenceTerm.conceptSource.name === 'ICD-10-WHO');
+export function savePatientDiagnoses(abortController: AbortController, payload: DiagnosisPayload) {
```

Comment:
Since this only save one diagnosis at a time...

```suggestion
export function savePatientDiagnosis(abortController: AbortController, payload: DiagnosisPayload) {
```

Reply:
Is not diagnoses the plural? Why should I want the singular here?

Reply:
Because this function only creates one diagnosis at a time.